Core.DomainModel/GDPR/DataProcessingRegistration.cs

```diff
@@ -340,6 +340,48 @@ public Maybe<IEnumerable<DataProcessingRegistrationOversightDate>> SetOversightC
         }
 
         public virtual ICollection<ItContract.ItContract> AssociatedContracts { get; set; }
+        public int? MainContractId { get; set; }
+        public virtual ItContract.ItContract MainContract { get; set; }
+        public bool IsActiveAccordingToMainContract => CheckContractValidity();
```

Comment:
let's make the validation model open to changes - I suspect this is just the first parameter of many so no need to make it too simple.. so let's make it like the other two validations